library/cc/request_headers_builder.cc

```diff
@@ -28,6 +28,11 @@ RequestHeadersBuilder::addUpstreamHttpProtocol(UpstreamHttpProtocol upstream_htt
   return *this;
 }
 
+RequestHeadersBuilder& RequestHeadersBuilder::addKeyValue(std::string key, std::string value) {
```

Comment:
nit: I'd call this`set`, for alignment with the other builders: https://github.com/envoyproxy/envoy-mobile/blob/main/library/kotlin/io/envoyproxy/envoymobile/RequestHeadersBuilder.kt#L43